manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.18.yaml

```diff
@@ -39,7 +39,10 @@ spec:
             type_url: type.googleapis.com/stats.PluginConfig
             value:
               {{- if not .Values.telemetry.v2.prometheus.configOverride.outboundSidecar }}
-              {}
+              {
+                "graceful_deletion_interval": "1s",
```

Comment:
I think envoyfilter is not going to be used for most tests anymore so this may not be giving much coverage

Reply:
@kyessenov we can introduce some feature flags into istiod before put them into Telemetry API

Reply:
Yeah, I figured. Would be good to have some "DISABLE_ENVOY_FILTER" flag so that it's clear what depends on it.
@zirain I see you updated the proto, will wait for it to sync before retrying.

Reply:
FYI: https://github.com/istio/api/pull/2769